### PR TITLE
Added comments for some methods

### PR DESCRIPTION
**Pull request by tasneemkoushar**

### CODE REVIEW

File: src/common/Connection.ts [hunk 1]
```diff
@@ -263,6 +263,9 @@ export class CDPSession extends EventEmitter {
     this._sessionId = sessionId;
   }
 
+  /**
+   * @internal
+   */
```
Comment (jackfranklin) [inline src/common/Connection.ts]: I don't think this is internal, as it's documented here: https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-cdpsessionconnection

Could we update this PR to add documentation for this method, please?

File: src/common/Page.ts [hunk 2]
```diff
@@ -2438,26 +2439,26 @@ export class Page extends EventEmitter {
...
+   *  - `width` : <number> width of clipping area.</number><br/>
+   *  - `height` : <number> height of clipping area.</number>
    *
    * - `omitBackground` : <boolean> Hides default white background and allows
-   *   capturing screenshots with transparency. Defaults to `false`
+   *   capturing screenshots with transparency. Defaults to `false`</boolean>
    *
    * - `encoding` : <string> The encoding of the image, can be either base64 or
-   *   binary. Defaults to `binary`.
+   *   binary. Defaults to `binary`.</string>
```
Comment (jackfranklin) [inline src/common/Page.ts]: all these closing tags are incorrect - but I think what we should do is get rid of the `<string>` / `<number>` part of the docs entirely.

**Review by jackfranklin — COMMENTED** [review]
Thanks - left some comments!

File: src/common/Connection.ts (hunk 1 above)
Comment (tasneemkoushar) [reply src/common/Connection.ts]: Sure, I will mark it as public

File: src/common/Page.ts (hunk 2 above)
Comment (tasneemkoushar) [reply src/common/Page.ts]: Ok I will omit these invalid tags

File: src/common/Connection.ts [hunk 3]
```diff
@@ -48,7 +48,7 @@ export const ConnectionEmittedEvents = {
 } as const;
 
 /**
- * @internal
+ * @public
```
Comment (jackfranklin) [inline src/common/Connection.ts:51]: we should also add TSDoc comments to the methods on this class that are documented on pptr.dev (https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-cdpsessionconnection). Could you look at that in a future PR, please?

**Review by jackfranklin — APPROVED** [review]
Thanks!

**Pull request merged**